### ISSUE TITLE
About IntelWifi (native)

### ISSUE THREAD

**Issue by mehmetyukcell** [DominikHackintosh/HP-EliteBook-840-G4-Hackintosh-Big-Sur#4]
Dear DominikHackintosh, are you used IntelWifi 8260 (native) airdrop airport handoff?

**Comment by DominikHackintosh** [comment]
yes i use this card, handoff is working, airdrop i could not try because do not use iphones

**Comment by mehmetyukcell** [comment]
Thanks for your comment. Also, my intel wifi aspm shows disabled? how to it enabled?
![Ekran Resmi 2020-11-12 16 23 28](https://user-images.githubusercontent.com/62822060/98945680-e5796580-2503-11eb-9bb4-c1f2fbc312c4.png)

**Comment by DominikHackintosh** [comment]
what system you boot and which kext you use?
![Screenshot 2020-11-12 at 14 37 44](https://user-images.githubusercontent.com/71799198/98946869-bdced100-24f4-11eb-9d88-7631d9cd25aa.png)

**Comment by mehmetyukcell** [comment]
MY system Hp probook 450 G6 (i7 8565u) Big sur with opencore

**Comment by mehmetyukcell** [comment]
Kext AirPort itlwm on the intel 9560ac wifi

**Comment by DominikHackintosh** [comment]
definitely should work, take care that you use the right one for big sur, try to delete all of your network configurations. if this does not work, then try itlwm.kext with heliport

https://github.com/OpenIntelWireless/itlwm/releases/tag/v1.2.0-alpha

**Comment by DominikHackintosh** [comment]
you can try by deleting /Library/Preferences/SystemConfiguration/networkinterfaces.plist and com.apple.airport.preferences.plist then reboot

**Comment by mehmetyukcell** [comment]
I already use it exactly like that (Final kext airportitlwm 1.2.2). Only, Hackintool Shows disable aspm for intel wifi, but works too.

**Comment by DominikHackintosh** [comment]
ok fine, for bluetooth you need this https://github.com/OpenIntelWireless/IntelBluetoothFirmware

**Comment by mehmetyukcell** [comment]
https://github.com/OpenIntelWireless/itlwm/releases/tag/v1.2.0-alpha new release

**Comment by DominikHackintosh** [comment]
yes i know they update nearly every second day now ;)

**Comment by cyberfunk** [comment]
> Dear DominikHackintosh, are you used IntelWifi 8260 (native) airdrop airport handoff?

AirDrop is not working for me. My Hackintosh shows up on my iPhone but I can't send anything. My iPhone does not show up on my Hackintosh. Please let me know if you have any luck getting it activated with a successful transfer.

**Comment by DominikHackintosh** [comment]
always try the newest versions, i cannot test it because i have no iphone here

**Comment by cyberfunk** [comment]
> always try the newest versions, i cannot test it because i have no iphone here

The latest is very old 1.1.0 stable. 1.2.0 is still in Alpha and they suggest we don't use it. I think the last time I tried an Alpha build it didn't work at all and I had to go back to 1.1.0. Will give the latest Alpha a try.

**Comment by DominikHackintosh** [comment]
i always use the alpha version with no kind of problems

**Comment by cyberfunk** [comment]
I installed the latest Alpha 1.2.0, cleared kextcache and rebooted. No luck with AirDrop it just won't transfer from iPhone to Hackintosh even though it shows up on the iPhone. It just says waiting... endlessly. Even if I hold the phone right next to the laptop.

**Comment by DominikHackintosh** [comment]
did you try it with itlwm and heliport?

**Comment by cyberfunk** [comment]
I just did a thorough Google search and every GitHub and other forum said it's still in the works so no AirDrop for now. Still I'm super impressed that they brought Intel Wireless to Hackintosh so I'm not complaining😁

**Comment by DominikHackintosh** [comment]
we will see, they are working on it, nearly every second day there is another update. really a nice project ;)

**Comment by GGorAA** [comment]
I also want to say my 2 cents,

I am using the Airportilwm.kext for enabling Intel Wireless. Works good so far :D

I installed for now 1.2.0 alpha, and it works great! But it has some bugs (obviously, this is alpha), but I update that kext every time!

**Comment by DominikHackintosh** [comment]
![124902322_2788114108172667_2408735150919830509_o](https://user-images.githubusercontent.com/71799198/99154651-c9b3c200-26b1-11eb-9dc2-b264c5ab8c1d.jpg)
this picture i got, the progress seems to go on, but file transfer was not possible yet

**Comment by mehmetyukcell** [comment]
Where installed Airportitlwm? L/E or Opencore Kexts inject

**Comment by DominikHackintosh** [comment]
do it in openCore Kext Folder and add to plist
